src/lsptoolshost/roslynLanguageServer.ts

```diff
@@ -412,6 +413,10 @@ export class RoslynLanguageServer {
             args.push('--logLevel', logLevel);
         }
 
+        if (this.additionalExtensionPaths.length > 0) {
+            args.push('--additionalExtensions', `"${this.additionalExtensionPaths.join(',')}"`);
```

Comment:
Why comma delimited? I'd expect each switch passed separately, since this will break with paths that contain commas.

Reply:
I can do that instead